scripts/import/dbSNP_v2/update_new_sets.pl

```diff
@@ -0,0 +1,340 @@
+#!/usr/bin/env perl
+use strict;
+use DBI;
+use Socket;
+use Bio::EnsEMBL::Registry;
+use Getopt::Long;
+use POSIX qw(strftime);
+use Cwd qw(cwd);
+
+$| = 1;
+socketpair(CHILD, PARENT, AF_UNIX, SOCK_STREAM, PF_UNSPEC) or die "ERROR: Failed to open socketpair: $!";
+PARENT->autoflush(1);
+
+my $registry = 'Bio::EnsEMBL::Registry';
+my $config = configure();
+
+debug($config, "Importing variation set starting now");
+
+my $reg_file = $config->{registry};
+$registry->load_all($reg_file);
+my $db_adaptor = $registry->get_DBAdaptor("homo_sapiens", "variation");
+my $dbh = $db_adaptor->dbc;
+my $dbname;
+$dbname = $dbh->dbname;
+debug($config, "Connected to the new database $dbname");
+
+my $TMP_DIR = $config->{tmp};
+
+my $new_vf_file = "variation_feature.txt";
+my $tmp_vset = "variation_name_set.txt";
+my $tmp_merged = "new_variation_set_variation.txt";
+my $tmp_vs_file = "vset_concat.txt";
+my $old_dbh;
+my $old_reg_file = $config->{old_registry} if (defined ($config->{old_registry}));
+my $release =  $config->{release} if (defined ($config->{release}));
+
+if ($old_reg_file) {
+  $registry->load_all($old_reg_file);
+  $db_adaptor = $registry->get_DBAdaptor("homo_sapiens", "variation");
+  # Connect to DBI
+  $old_dbh = $db_adaptor->dbc;
+  $dbname = $old_dbh->dbname;
+  debug($config, "Connected to the old database $dbname");
+} else {
+  my $old_release = $release - 1;
+  my $old_dbname = $dbh->dbname =~ s/_${release}_/_${old_release}_/gr;
+  my $old_host = $dbh->host;
+  my $old_port = $dbh->port;
+  #Connect to DBI
+  $old_dbh = DBI->connect("DBI:mysql:database=${old_dbname};host=${old_host};port=${old_port}", "ensro", "");
+  debug($config, "Connected to the old database $old_dbname");
+}
+
+debug($config, "Selecting minimum and maximum variation feature");
+my $sql = qq{ SELECT MIN(variation_feature_id), MAX(variation_feature_id) FROM variation_feature};
+my $sth = $old_dbh->prepare($sql);
+my $sth = $dbh->prepare($sql);
```

Comment:
Is the query supposed to run in the old db or the new? 
Here it's calling both but in the end, it runs in the new db.